.github/workflows/event-gateway-release.yml

```
@@ -0,0 +1,96 @@
+          go-version: '1.26.1'
+          cache: false
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v3
+        with:
+          platforms: linux/amd64,linux/arm64
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+        with:
+          install: true
```

actionlint (1.7.12)

[error] avoid using deprecated input "install" in action "docker/setup-buildx-action@v3": "docker buildx install" command is deprecated and will be removed in a future release, use BUILDX_BUILDER environment variable instead

(action)